integration_tests/utils.py

```diff
@@ -13,6 +13,21 @@ def get_repo_root_path():
     return os.path.dirname(rust_vmm_ci_path)
 
 
+def get_coverage_config_path(platform):
+    target_file = f"coverage_config_{platform}.json"
+    dirs = [os.getcwd()]
```

Comment:
Let's add a comment here why we are not using os.walkdir so that in the future people don't try to wrongly optimize it.

Reply:
Done.